cmstestsuite/TestCleanCheckout.py

```diff
@@ -77,7 +77,8 @@ def setup_cms():
     os.environ["PYTHONPATH"] = "%(TEST_DIR)s" % CONFIG
 
     info("Building cms.")
-    sh("./setup.py build")
+    sh("./prerequisites.py compile_l10n")
+    sh("./prerequisites.py compile_isolate")
```

Comment:
> Why do you say you'd have 3 commands instead of 2?

Because you would need to do:

    ./setup.py <command_to_compile_isolate_and_l10n>
    ./prerequisites.py install
    ./setup.py install